src/app/cakes/cake.ts

```diff
@@ -0,0 +1,29 @@
+export interface CakeInterface {
```

Comment:
As a good practice, you can add the letter I before the name of your interfaces, to identify them versus the classes. Ex. `IBread` or `iBread`.

Since we are using a Single File Responsibility file distribution, this interface should live in a separate file named `cake-status.interface.ts`.